client/asset/dcr/dcr.go

```diff
@@ -632,21 +639,29 @@ func (dcr *ExchangeWallet) feeRate(confTarget uint64) (uint64, error) {
 	return 1 + uint64(atomsPerKB)/1000, nil // dcrPerKB * 1e8 / 1e3
 }
 
-// feeRateWithFallback attempts to get the optimal fee rate in atoms / byte via
-// FeeRate. If that fails, it will return the configured fallback fee rate.
-func (dcr *ExchangeWallet) feeRateWithFallback(confTarget, feeSuggestion uint64) uint64 {
+// targetFeeRateWithFallback attempts to get a fresh fee rate for the target
+// number of confirmations, but falls back to the suggestion or fallbackFeeRate
+// via feeRateWithFallback.
+func (dcr *ExchangeWallet) targetFeeRateWithFallback(confTarget, feeSuggestion uint64) uint64 {
 	feeRate, err := dcr.feeRate(confTarget)
 	if err == nil {
 		dcr.log.Tracef("Obtained local estimate for %d-conf fee rate, %d", confTarget, feeRate)
 		return feeRate
 	}
-	if feeSuggestion > 0 && feeSuggestion < dcr.fallbackFeeRate && feeSuggestion < dcr.feeRateLimit {
-		dcr.log.Tracef("feeRateWithFallback using caller's suggestion for %d-conf fee rate, %d. Local estimate unavailable (%q)",
-			confTarget, feeSuggestion, err)
+	dcr.log.Tracef("no %d-conf feeRate available: %v", confTarget, err)
```

Comment:
I think this trace can go away.  There will be a log in either branch of `feeRateWithFallback`.
The logs have just gotten a bit noisier is all.  e.g.

```
2022-02-02 13:58:25.501 [DBG] CORE: failed to get fee suggestion from dcr FeeRater: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 13:58:25.503 [TRC] CORE[dcr]: no 1-conf feeRate available: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 13:58:25.503 [TRC] CORE[dcr]: feeRateWithFallback using caller's suggestion for fee rate, 10. Local estimate unavailable
2022/02/02 13:58:25 "POST http://127.0.0.3:5758/api/maxbuy HTTP/1.1" from 127.0.0.1:42018 - 200 204B in 55.946921ms
2022-02-02 13:58:26.347 [DBG] CORE: failed to get fee suggestion from dcr FeeRater: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 13:58:26.349 [TRC] CORE[dcr]: no 1-conf feeRate available: -9: RPC passthrough requires dcrd RPC synchronization
2022-02-02 13:58:26.349 [TRC] CORE[dcr]: feeRateWithFallback using caller's suggestion for fee rate, 10. Local estimate unavailable
2022/02/02 13:58:26 "POST http://127.0.0.3:5758/api/preorder HTTP/1.1" from 127.0.0.1:42018 - 200 1200B in 55.668948ms
```